docs/high-level-design.md

```diff
@@ -74,7 +74,7 @@ address, following issues (not an exhaustive list):
 - [**CSP (content security policy)**](https://csp.withgoogle.com/docs/index.html) -
   e.g. by automatically adding script nonces to HTML responses, adding relevant
   security headers
-- **Transport Security** - e.g. by enforcing HSTS support
+- **Transport Security** - e.g. by [enforcing HSTS support](https://github.com/google/go-safeweb/blob/master/docs/plugins/hsts.md)
```

Comment:
This link is not working for me.

Reply:
It will work once this is merged.

Reply:
Are relative links possible?

Reply:
Yup, https://github.blog/2013-01-31-relative-links-in-markup-files/
Fixed!